res/skins/Tango/deck_overview.xml

```diff
@@ -26,6 +26,8 @@ Variables:
     <PlayPosColor>#FF4300</PlayPosColor>
     <PlayedOverlayColor><Variable name="PlayedOverlayColor"/></PlayedOverlayColor>
     <EndOfTrackColor><Variable name="EndOfTrackColor"/></EndOfTrackColor>
+    <LabelTextColor>#ffffff</LabelTextColor>
+    <LabelBackgroundColor>#85000000</LabelBackgroundColor>
```

Comment:
I'm not sure I like this. The waveform underneath is not as visible.

Reply:
IMO the default opacity of the label's box is way to low.
The labels should add information not noise..
As you can see: as soon as a loop region comes into play the labels are difficult to read.

this PR:
![image](https://user-images.githubusercontent.com/5934199/65552768-89313c00-df25-11e9-9118-ef632ac3c00d.png)

default opacity:
![image](https://user-images.githubusercontent.com/5934199/65552733-6f8ff480-df25-11e9-8503-f92da8d1f5e5.png)

Deere default:
![image](https://user-images.githubusercontent.com/5934199/65552790-9c440c00-df25-11e9-9d21-d8974a5429ec.png)

Reply:
> as soon as a loop region comes into play the labels are difficult to read

You're right; that was not a use case I considered.

Reply:
ok. I didn't check this in the other skins. I'll do so and supply another PR later